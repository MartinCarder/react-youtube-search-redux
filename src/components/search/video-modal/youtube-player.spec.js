import React from 'react'
import { render } from 'enzyme'
import YoutubePlayer from './youtube-player'

describe('<YoutubePlayer>', () => {
  it('component renders', () => {
    const wrapper = render(<YoutubePlayer />)
    expect(wrapper).toMatchSnapshot()
  })
})

// TODO test player mounting script
