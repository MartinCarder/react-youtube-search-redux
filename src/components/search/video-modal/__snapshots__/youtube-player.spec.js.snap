// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<YoutubePlayer> component renders 1`] = `
.c0 {
  width: 100%;
  height: 100%;
}

<div
  class="c0"
>
  <div>
    
  </div>
</div>
`;
